.azure-pipelines/azure-pipelines-linux-clang.yml

```diff
@@ -29,6 +29,9 @@ jobs:
           llvm_version: '9'
         clang_10:
           llvm_version: '10'
+        clang_10_avx512:
+          llvm_version: '10'
+          avx512: 1
     pool:
```

Comment:
There's alreay a configuration for testing avx512 with gcc, so I'm not sure that adding a new one is required, not that it will actually work.

Reply:
gcc and clang have a different set of warning. But yeah, that doesn't seem to be the problem here.